less/about/preferences.less

```diff
@@ -455,6 +455,46 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
```

Comment:
yes...

Add Funds should be "Add funds..." and use smaller text. Button should match height of budget menu. Said menu should move up to align with "Add funds..." button.